src/services/authentication/strategies/ApiKeyStrategy.js

```diff
@@ -8,8 +8,9 @@ class ApiKeyStrategy extends AuthenticationBaseStrategy {
 			return {
 				strategy: this.name,
 				apiKey: req.headers['x-api-key'],
+				// todo: remove route when there are permissions for api-keys
 				// eslint-disable-next-line no-underscore-dangle
-				route: req._parsedUrl.path.split('/')[1], // todo: remove this when there are permissions for api-keys
+				route: req._parsedUrl.path.split('/')[1].split('?')[0],
```

Comment:
there is no evidence for the existence of a `[1]`st element, but a `[0]`st element always exist for strings...

Reply:
der url path beginnt immer mit "/", daher gibt es immer ein [1] element in diesem fall.